### ISSUE TITLE
docs.rs failed to build nannou-0.12.0

### ISSUE THREAD

**Issue by p3zo** [nannou-org/nannou#429]
The [0.12.0 docs.rs page](https://docs.rs/crate/nannou/0.12.0) shows a build failure from 11/6/19.

The [failed build logs](https://docs.rs/crate/nannou/0.12.0/builds/196525) show the error `OSError: [Errno 30] Read-only file system` .

The build container used these package versions:
```
rustc 1.40.0-nightly (1423bec54 2019-11-05)
docsrs 0.6.0 (2cc016f 2019-11-06)
```

**Comment by mitchmindtree** [comment]
Thanks for the report!

I reported this to docs.rs [a little while back](https://github.com/rust-lang/docs.rs/issues/471#event-2820463675) - it seems the issue is to do with one of our dependencies attempting to write to the filesystem in a way that the docs.rs build sandbox does not like. Apparently this issue has been addressed in the latest version of the culprit dependency, so it looks like this will be addressed once the necessary chain of dependencies is updated to `xcb` version `0.9`.

**Comment by mitchmindtree** [comment]
For those who would like to see some rendered docs:

- the `0.11.1` docs are [available here](https://docs.rs/nannou/0.11.1/nannou/) and are almost identical to `0.12`.
- Otherwise, if you would like 0.12, you can build the docs locally and open them in your browser with `cargo doc --open`.

**Comment by tpltnt** [comment]
Some people I know where irritated by the missing (current) documentation. Is there any roadmap on when this issue will be fixed?

**Comment by mitchmindtree** [comment]
Judging by the discussion in rust-lang/docs.rs#471 the crate responsible for breaking the build on docs.rs has been fixed and 0.12.1 (or 0.13 depending on what comes first) should build and upload successfully next time we publish. From the looks of it I don't think the docs.rs team will try re-building 0.12 again.

**Comment by tpltnt** [comment]
Fixed by shipping nannou 0.13.1 ?
